data/xml/2018.eamt.xml

```diff
@@ -0,0 +1,529 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2018.eamt">
+  <volume id="main" ingest-date="2023-05-05">
+    <meta>
+      <booktitle>Proceedings of the 21st Annual conference of the European Association for Machine Translation</booktitle>
```

Comment:
"Conference" should be capitalized. Actually, I see that it is lowercased on [many other EAMTs](https://aclanthology.org/venues/eamt/)—can you fix those in this PR?

Reply:
Done